java/open-mobile-scene-package/README.md

```diff
@@ -2,19 +2,26 @@
 
 Opens and displays a scene from a Mobile Scene Package (.mspk).
 
-![Open Mobile Scene Package App](open-mobile-scene-package.png)
+![Image of open mobile scene package](open-mobile-scene-package.png)
+
+## Use case
+
+An .mspk file is an archive containing the data (specifically, basemaps and features), used to display an offline 3D scene.
 
 ## How to use the sample
 
-Run the app and allow read permissions.
+When the sample opens, it will automatically display the Scene in the Mobile Map Package.
+
+Since this sample works with a local .mspk, you may need to download the file to your device.
```

Comment:
For Android, we *will* have to download the file. Let's make that more specific here.